website/docs/cli/cloud/settings.mdx

```diff
@@ -90,11 +90,11 @@ Use the following environment variables to configure the `cloud` block:
 
 
 - `TF_ORGANIZATION` - The name of the organization. Serves as a fallback for `organization`
-    in the cloud configuration. If both are specified, the configuration takes precedence. 
+    in the cloud configuration. If both are specified, the configuration takes precedence.
 
-- `TF_HOSTNAME` - The hostname of a Terraform Enterprise installation. Serves as a fallback if `hostname` is not specified in the cloud configuration. If both are specified, the configuration takes precendence. 
+- `TF_HOSTNAME` - The hostname of a Terraform Enterprise installation. Serves as a fallback if `hostname` is not specified in the cloud configuration. If both are specified, the configuration takes precendence.
 
-- `TF_WORKSPACE` - The name of a single Terraform Cloud workspace.  If the `workspaces` attribute is not included in your configuration file, the `cloud` block interprets `TF_WORKSPACE` as the `name` value of the `workspaces` attribute. The workspace must exist in the organization specified in the configuration or `TF_ORGANIZATION`. If the `cloud` block uses tags, Terraform Cloud will return an error if the value of `TF_WORKSPACE` is not included in the set of tags. Refer to [TF_WORKSPACE](https://www.terraform.io/cli/config/environment-variables#tf_workspace) for more details.
+- `TF_WORKSPACE` - The name of a single Terraform Cloud workspace.  If the `workspaces` attribute is not included in your configuration file, the `cloud` block interprets `TF_WORKSPACE` as the `name` value of the `workspaces` attribute. The workspace must exist in the organization specified in the configuration or `TF_ORGANIZATION`. You can set this variable if the `cloud` block in your configuration uses tags. However, Terraform Cloud will return an error if the value of `TF_WORKSPACE` is not included in the set of tags. Refer to [TF_WORKSPACE](https://www.terraform.io/cli/config/environment-variables#tf_workspace) for more details.
```

Comment:
This is really minor, but we often use "`cloud` configuration" and "`cloud` block" interchangeably (which may lead to confusion). I think now that we're revisiting this we should stick with one or the other (my vote is for `cloud` configuration)

Reply:
I'm wondering if it's worth mentioning somewhere that `TF_WORKSPACE`, in the context of the `cloud` configuration, does not create a new workspace as one would expect using `TF_WORKSPACE`.

Reply:
ohhh this is good feedback!

1. cloud configuration feels a bit vague to me - you are right though, we're NOT consistent with it. To me, "cloud configuration" could be a stand in for well, configuring any cloud provider. And so I think it might be better to be very explicit "`cloud` block in your configuration". Thoughts? Once we agree I will go back and fix this inconsistency - thank you so much for flagging this, it's a great catch!

2. Yes - I think that is important to mention! okay so - just to check - if you specify a workspace name that does not yet exist it does NOT get automatically created. Does TFC throw an error in that case? I can add a sentence in about that as well :-)

Reply:
On point 1:  thinking about it a bit more, you're totally correct! I think "`cloud` block in your configuration" might be the way to go here. _However_, given that this content lives in the page `Terraform Cloud Settings`, under the `cloud block` section, we could also just trim it and say `cloud` block. 

On point 2: TFC won't throw an error; we simply added those validation steps when the `cloud` block gets configured. So if the validation fails, Terraform never gets to reach the logic where it handles creating `TF_WORKSPACE` should it need to. Furthermore, these validations only happen when `TF_WORKSPACE` is set and the intent is to use it to configure the `cloud` block, meaning `name` is omitted or the `tags` attribute is set.

Reply:
I like that regarding `cloud` block --> let's do that!  And okay - thank you for clarifying!